Cartridges/int_checkoutcom_sfra/cartridge/client/default/js/checkout/address.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+/**
+ * Populate the Billing Address Summary View
+ * @param {string} parentSelector - the top level DOM selector for a unique address summary
+ * @param {Object} address - the address data
+ */
+function populateAddressSummary(parentSelector, address) {
+    $.each(address, function (attr) {
+        var val = address[attr];
+        $('.' + attr, parentSelector).text(val || '');
+    });
```

Comment:
i don't think we should add this file to our integration. Please can you look into removing this file also.

Reply:
only add this file if it's removal will negatively affect our cartridge.